### PR TITLE
532 bug enum4linux tooltip grammar

### PR DESCRIPTION
**Pull request by anjojoseph**
Hi,
532 bug enum4linux tooltip grammar task is completed. 
Kindly check and merge the branch.
Thank you.

### CODE REVIEW

File: src/components/Enum4Linux/Enum4Linux.tsx
Comment (bradleybogg) [inline src/components/Enum4Linux/Enum4Linux.tsx:1]: Hi @anjojoseph,

These changes are very good and do not technically need further editing. I will ask for some changes though just to bring in line with other tool tips.

Very good work. After the below changes please look for a task with a 'medium' difficulty. This is to ensure you start making evidence for at least a passing grade.

File: src/components/Enum4Linux/Enum4Linux.tsx [hunk 1]
```diff
@@ -9,20 +9,20 @@ import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFil
...
 const description_userguide =
-    "Enum4linux is a tool used for the enumeration of information from Windows and Samba operating systems. " +
+    "Enum4linux is a tool used for enumerating information from Windows and Samba operating systems. " +
     "It is particularly useful for identifying the remote OS of a system and providing a list of the users " +
     "and group memberships found within the system.\n\nOptions for the tool can be found at: " +
     "https://www.kali.org/tools/enum4linux/\n\n" +
     "Using Enum4Linux:\n" +
-    "Step 1: Enter a Target IP address.\n" +
+    "Step 1: Enter a target IP address.\n" +
     "       Eg: 192.168.1.1\n\n" +
```
Comment (bradleybogg) [inline src/components/Enum4Linux/Enum4Linux.tsx:18]: Make in line with line 17. Remove \n\n after the example and simply have it read:

Step 1: Enter a target IP address. E.g 192.168.1.1\n" +

File: src/components/Enum4Linux/Enum4Linux.tsx [hunk 2]
```diff
@@ -9,20 +9,20 @@ import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFil
...
-    "Step 2: Enter an Option for the Enumeration.\n" +
+    "Step 2: Enter an option for the enumeration.\n" +
     "       Eg: U (get userlist)\n\n" +
```
Comment (bradleybogg) [inline src/components/Enum4Linux/Enum4Linux.tsx:20]: Make in line with line 19. Remove \n\n after the example and have it simply read:

"Step 2: Enter an option for the enumeration. E.g U (get userlist)\n" +

File: src/components/Enum4Linux/Enum4Linux.tsx [hunk 3]
```diff
@@ -9,20 +9,20 @@ import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFil
...
-    "Step 3: Enter any Parameters.\n" +
+    "Step 3: Enter any parameters.\n" +
     "       Eg: example.txt\n\n" +
```
Comment (bradleybogg) [inline src/components/Enum4Linux/Enum4Linux.tsx:22]: Make in line with line 21. Remove \n\n after the example and have it simply read:

"Step 3: Enter any parameters. E.g example.txt. \n" +

**Review by bradleybogg — CHANGES_REQUESTED** [review]

**Comment by bradleybogg** [comment]
Requested changes not implemented. Please re-raise when completed.

**Comment by anjojoseph** [comment]
Changes has been done. opened pull request : 532 bug enum4linux tooltip grammar #639

**Pull request closed without merging**